frontend/src/metabase/visualizations/components/settings/ChartSettingsListColumns/utils.ts

```diff
@@ -0,0 +1,38 @@
+import type { Column } from "metabase-types/types/Dataset";
+
+import {
+  isAvatarURL,
+  isEntityName,
+  isCategory,
+  isDescription,
+  isDate,
+  isNumeric,
+  isPK,
+} from "metabase-lib/types/utils/isa";
+
+const getIdOrRef = (column: Column | null) => column?.id ?? column?.field_ref;
+
+const findA = (columns: Column[], predicate: (column: Column) => boolean) => {
+  return columns.find(predicate);
+};
+
+export const getDefaultColumns = (columns: Column[]) => {
+  const findInColumns = (predicate: (column: Column) => boolean) =>
+    findA(columns, predicate);
```

Comment:
nit: but feels like we can just use `columns.find(predicate)` instead of `findA`?